source/_integrations/devolo_home_network.markdown

```diff
@@ -132,6 +132,96 @@ This integration only supports using the API the devolo Home Network App uses. T
 
 The devolo Gigabridge is the only device that comes with a default password. However, it seems that in factory default the password works for the device website but not for the API. If you give the device a new password via the website, it is applied to both and the integration starts working. Even using the same password again works.
 
+## Example automations
+
+### Restart PLC device on loss of pairing
+
+PLC networks are sometimes flaky. To restore a network's state it's sometimes a good idea to reboot the PLC device attached to the router, if the number of PLC devices is lower as expected. If you apply this automation, keep in mind, that devices might be expectedly on standby. In this example, the expected number of devices is 3.
+
+{% raw %}
+
+```yaml
+alias: PLC Feeder Restart
+description: "Restart device connected to the router if number of PLC devices is unexpected low"
+triggers:
+  - trigger: numeric_state
+    entity_id:
+      - sensor.YOUR_DEVICE_connected_plc_devices  # Replace with your device's sensor
+    for:
+      hours: 0
+      minutes: 10
+      seconds: 0
+    below: 3
+conditions: []
+actions:
+  - device_id: DEVICE_ID  # Replace with your device's ID
+    domain: button
+    entity_id: ENTITY_ID  # Replace with your restart button's entity ID
+    type: press
+mode: single
+```
+
+{% endraw %}
+
+### Notify on data rate drop
+
+Noise on the electric wire can significant disturb PLC data rates. A notification close to a drop can help identify the action that lead to the drop. The following example takes 25% as threshold.
+
+{% raw %}
+
+```yaml
+alias: PLC Data rate
+description: PLC Data rate dropped more than 25%
+triggers:
+  - entity_id:
+      - sensor.YOUR_DEVICE_plc_downlink_phy_rate_TARGET  # Replace with your device's sensors
+      - sensor.YOUR_DEVICE_plc_uplink_phy_rate_TARGET
+    trigger: state
+conditions:
+  - condition: template
+    value_template: >-
+      # Checks if new value is less than 75% of previous value
+      {{ not(0.75 < (trigger.to_state.state|float /
+      trigger.from_state.state|float)) }}
```

Comment:
Can we make this more readable

Reply:
Don't ask why I made it that complicated, because I don't know myself ...